docs/modules/ROOT/pages/developer-guide.adoc

```diff
@@ -10,7 +10,7 @@ to activate the virtual environment.
 
 Next, run:
 ----
-pip install --editable .
+pip install --editable
```

Comment:
This one is not a spelling mistake, the `.` indicates that you want to install the current directory

Reply:
Done, reversed it